cake/banner.cake

```diff
@@ -0,0 +1,15 @@
+// Banner class displays banners similar to those created by Cake
+// We use this to display intermediate steps within a task
+public static class Banner
+{
+    public static void Display(string message, char barChar='-', int length=0)
+    {
+        if (length == 0) length = message.Length;
```

Comment:
If message has multiple lines this value will be incorrect.

Reply:
Good point. I don't use it that way but somebody could.